common/io/ExtendedSerial.cpp

```diff
@@ -36,6 +36,15 @@
 
 #ifdef HAVE_ASM_TERMIOS_H
 // use this not standard termios for custom baud rates
+//
+// On mips architectures, <asm/termios.h> sets some cpp macros which cause
+// <cerrno> (included by <ostream>, used by <ola/Logging.h>) to not define
+// ERANGE, EDOM, or EILSEQ, causing a spectacular compile failure there.
+//
+// Explicitly include <cerrno> now to avoid the issue.
+#if HAVE_ERRNO_H
```

Comment:
As discussed on IRC, I wonder if we should die during configure if errno.h isn't present and then take these guards out here.